install.bat

```diff
@@ -0,0 +1,2 @@
+set GOPATH=C:\go-R
```

Comment:
is it smth local environment specific, i.e. binds other users to the particular GOPATH